Change selector to use button element instead of div

tschuyebuhl opened the pull request:
as per title, broke it in my last commit >_<

mkaczanowski commented:
> as per title, broke it in my last commit >_<

What commit? Also where is testplan (screenshot would do to understand changes etc)

tschuyebuhl commented:
> What commit?
here: https://github.com/ChorusOne/blazar/commit/d0e6420fd58752220b3faeeb3988796cefc3ed12#diff-09f36a8c9212ee96b23c202e2cd29843aa4ecdc43b8fb113119fcc9a1bc005e3R406

In the last commit, I changed the element type from div to a button, and broke the jQuery selector. While the selector worked, this also introduced another issue which was duplicated element identifiers. This _should_ fix it. 

Also, edited the PR to include the test plan. Thank you for the heads up

mkaczanowski commented:
![image](https://github.com/user-attachments/assets/7187de63-adf0-49ba-bb8e-dbbf37a8de86)

My eyesight is not getting better with age, but I can't see the test plan, nor the explanation what the PR does in the PR description :)

tschuyebuhl commented on internal/pkg/static/templates/index/index-blazar.html:
```diff
@@ -12,6 +12,23 @@
+              behavior: "smooth"
+            });
+          }
+        }
```
done! thank you
